src/spikeinterface/preprocessing/unsigned_to_signed.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+from ..core.core_tools import define_function_from_class
+from .basepreprocessor import BasePreprocessor, BasePreprocessorSegment
+from .filter import fix_dtype
+
+
+class UnsignedToSignedRecording(BasePreprocessor):
+    """
+    Converts a recording with unsigned traces to a signed one.
```

Comment:
Maybe a note here about @cwindolf comment that this will work on the way some people expect and point to the other function that does?

Reply:
good idea! done